Watchman.Discord/Areas/Responses/Services/ResponsesService.cs

```diff
@@ -28,7 +28,9 @@ public async Task<Response> GetResponseByOnEvent(string onEvent, ulong serverId
 
         public async Task AddResponse(string onEvent, string message, ulong serverId = 0)
         {
-            var addResponse = new Response(onEvent, message, serverId);
+            var query = new GetResponseQuery(onEvent);
+            var response = (await this._queryBus.ExecuteAsync(query)).Response;
+            var addResponse = new Response(onEvent, message, serverId, response.AvailableVariables);
```

Comment:
response.AvailableVariables

potencjalny null ref exception
nie masz pewności że to response już istnieje w bazie
nie bez powodu masz w parametrach domyślną wartość `ulong serverId = 0`